packages/shared/src/components/modals/award/GiveAwardModal.tsx

```diff
@@ -202,16 +196,15 @@ const CommentScreen = () => {
       { productId: product?.id, type, entityId: entity.id, note },
     ],
     mutationFn: award,
-    onSuccess: (result) => {
+    onSuccess: async (result) => {
       // TODO feat/transactions animation show award
```

Comment:
You can remove this now right?

Reply:
The comment?

Reply:
Yes

Reply:
There's still AS-1025 that does the animation